analysis/aggregate.py

```diff
@@ -69,6 +69,11 @@ def resample(event_counts, offset, func):
         event_counts.groupby(level=group_by)
         .resample(level=resample_by, rule=offset)
         .aggregate(func)
+        # Different aggregation functions behave differently when they are passed empty
+        # groups. For example, "sum" (`numpy.sum`) returns zeros; "mean" (`numpy.mean`)
+        # returns the missing value marker. We're resampling an irregular time series,
+        # so it's reasonable to replace zeros with the missing value marker.
```

Comment:
I think this comment (and the commit description) is backwards, isn't it? Your replacing missing value markers with zero, not the other way round. Or have I misunderstood what's going on here?

Reply:
You're quite correct. I'd originally tried going from zero to missing value marker, but found the code clunky. I'll update and force push.